example/lib/main.dart

```diff
@@ -10,12 +10,11 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Persian Fonts Example',
-      home: MyHomePage(),
-      theme: ThemeData(
-        textTheme: PersianFonts.vazirTextTheme,
-      )
-    );
+        title: 'Persian Fonts Example',
+        home: MyHomePage(),
+        theme: ThemeData(
+          textTheme: PersianFonts.vazirTextTheme,
+        ));
```

Comment:
Please don't change indention and leave it as it is, it probably happened because you use some code formatter.